packages/tools/src/utilities/cine/playClip.ts

```diff
@@ -298,16 +298,15 @@ function _stopClipWithData(playClipData) {
   }
 }
 
-function _getVolumeFromViewport(viewport): Types.IImageVolume {
-  const actorEntry = viewport.getDefaultActor();
+function _getVolumesFromViewport(viewport): Types.IImageVolume[] {
+  return viewport.getActors().map((actor) => cache.getVolume(actor.uid));
```

Comment:
there might be non volume actors, this should have a `.Boolean()` at the end to filter them